ExeterTeam25/group-software-project
# Responses points

Once a challenge is done by a user, it now cannot be done again and does not show on the main page. Profile model created and points added to this model when challenges are completed. New very simple page showing the logged in users completed challenges and total points.

How to test:

Create a new user and then by logging in as an admin, look at the profiles model to make sure it has been correctly generated with 0 points.

Complete a challenge, make sure it then disappears from the homepage and look in the my responses page to check the total points has increased and shows the completed challenge.
